lib/std/leb128.zig

```diff
@@ -233,8 +241,8 @@ test "deserialize signed LEB128" {
     try testing.expect((try test_read_ileb128(i8, "\xff\x7f")) == -1);
     try testing.expect((try test_read_ileb128(i16, "\xff\xff\x7f")) == -1);
     try testing.expect((try test_read_ileb128(i32, "\xff\xff\xff\xff\x7f")) == -1);
-    try testing.expect((try test_read_ileb128(i32, "\x80\x80\x80\x80\x08")) == -0x80000000);
-    try testing.expect((try test_read_ileb128(i64, "\x80\x80\x80\x80\x80\x80\x80\x80\x80\x01")) == @bitCast(i64, @intCast(u64, 0x8000000000000000)));
```

Comment:
perhaps we should add these 2 cases to the `expectError()` cases above to ensure this edge case is fully tested?

Reply:
Good shout!